gnomad_qc/v5/data_ingestion/federated_validity_checks.py

```diff
@@ -278,7 +306,17 @@ def validate_federated_data(
         metric_first_field=True,
         nhomalt_metric=nhomalt_metric,
     )
-    # TODO: consider adding check_global_and_row_annot_lengths, check for raw and adj.
+
+    if subsets:
+        logger.info("Comparing subset frequencies...")
+        compare_subset_freqs(
+            t=ht,
+            subsets=subsets,
+            verbose=verbose,
+            delimiter="_",
```

Comment:
should the delimiter default be "_" rather than "-" in the function? asking because it's called with an underscore here and in the tests

Reply:
not entirely sure, but left it to be consistent with previous versions